README.md

```diff
@@ -4,6 +4,17 @@
 
 The Java Debug Server is an implementation of Visual Studio Code (VSCode) Debug Protocol. It can be used in Visual Studio Code to debug Java programs.
 
+## Installation
+
+### Windows:
+```
+./mvnw.cmd clean install
+```
+### Linux and macOS:
+```
+./mvnw clean install
+```
+
```

Comment:
How about moving it behind the `Repository Structure` paragraph?

Reply:
Cool yeah! Sounds good.